src/components/RequestGroupComposition/Configuration.vue

```diff
@@ -501,7 +513,7 @@ export default {
   },
   data: function() {
     return {
-      show: true,
+      show: this.initialShow,
```

Comment:
is there a reason this can't just be parentShow?

Reply:
Yes. This is to enable us to be able to indicate whether we want to create new configuration panels as starting out as collapsed or not, which is useful when we generate a new mosaic and want to collapse all but the first Configuration, but want to keep things like the Request portion of the Request panel expanded. If we do something like 1) create new configurations initialized to the parent show value, which would be true, and then 2) collapse them, that would work in that it would look correct for the final rendered form, however, in that case we are paying the cost of fully rendering each configuration and all of the associated inputs up front, and then collapsing them, which defeats the purpose of this PR.